tests/unit/drop-zone.spec.js

```diff
@@ -160,9 +161,10 @@ describe('DropZone', () => {
     expect(w.vm.client.uppy.getState().files).toEqual({});
   });
 
-  test('run the drop zone in XHR mode', async () => {
+  test('change the uploader', async () => {
     expect(w.emittedByOrder().length).toBe(2);
-    w.setProps({ mode: 'XHR' });
+
+    w.setProps({ uploader: { uploaderClass: XHRUpload } });
```

Comment:
As already described above: when you change the endpoint here, the XHRUploader will not change the endpoint since it is not the first plugin that has been installed.